### PR TITLE
Updates to edit_attribute

### PR DESCRIPTION
**Pull request by smfreund**
(1) Correct typo in documentation, (2) add error message if user specifies either a missingValueCode or missingValueCodeExplanation but not both, (3) move return(eml) call to very end of the function, (4) add test to test file for missingValueCode error

### CODE REVIEW

**Comment by smfreund** [comment]
Review please at your convenience @isteves @maier-m

File: R/edit_attribute.R [hunk 1]
```diff
@@ -64,6 +64,12 @@ edit_attribute <- function(eml, dataTableNumber, attributeNumber, attributeName
     if(!is.null(formatString)) {
         attributeTable$formatString[attributeNumber] <- formatString
     }
+
+    if((is.null(missingValueCode) & !is.null(missingValueCodeExplanation)) |
```
Comment (isteves) [inline R/edit_attribute.R]: alternative: `length(c(missingValueCode, missingValueCodeExplanation)) == 1`

File: R/edit_attribute.R [hunk 2]
```diff
@@ -38,7 +38,7 @@ edit_attribute <- function(eml, dataTableNumber, attributeNumber, attributeName
                            missingValueCode = NULL, missingValueCodeExplanation = NULL){
 
     data <- EML::get_attributes(eml@dataset@dataTable[[dataTableNumber]]@attributeList)
-    attributeTable <- data.frame(data$attributes) #this excludes the factor table from enumerated domain.
+    attributeTable <-data$attributes #this excludes the factor table from enumerated domain.
 
     if(!is.null(attributeName)) {
```
Comment (isteves) [inline R/edit_attribute.R]: Something like this should work.  Not super neat, but certainly more concise:

```
attr_edits <- cbind(attributeName, attributeDefinition, domain,
      measurementScale, unit, numberType, definition, formatString,
      missingValueCode, missingValueCodeExplanation)

for(i in colnames(attr_edits)){
  attributeTable[,i][attributeNumber] <- attr_edits[,i]
}
```

**Comment by smfreund** [comment]
Incorporated new suggestions @isteves

**Pull request merged**